examples/reachable.myl

```diff
@@ -9,6 +9,6 @@ DO
                               EMIT Edge.dst AS addr]);
     Delta = DIFF(NewlyReachable, Reachable);
     Reachable = UNIONALL(Delta, Reachable);
-WHILE [*COUNTALL(Delta) > 0];
+WHILE [FROM COUNTALL(Delta) AS size EMIT *size > 0];
```

Comment:
If I understand the language change correctly, basically you only get the "super-duper-concise" `*x` if `x` is already an alias in the global state. You can't unbox a singleton in the emit clause that you're also defining in the emit clause.

I'm okay with that!